src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -4370,9 +4370,12 @@ Copyright (C) Microsoft Corporation. All rights reserved.
                                             Exclude="@(ReferenceCopyLocalPaths);@(_NETStandardLibraryNETFrameworkLib)" />
     </ItemGroup>
 
+    <ItemGroup Condition="'$(PublishSingleFile)' != 'true'">
+      <ClickOnceSatelliteAssemblies Include="@(IntermediateSatelliteAssembliesWithTargetPath);@(ReferenceSatellitePaths)" />
```

Comment:
Does this need to be "public" (not prefixed with `_`)? I ask since the prior one was "private".

Reply:
fixed.